ep/build-velox/src/build_velox.sh

```diff
@@ -104,6 +110,9 @@ function compile {
   if [ $ENABLE_HDFS == "ON" ]; then
     COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_HDFS=ON"
   fi
+  if [ $ENABLE_HDFS3 == "ON" ]; then
+    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_ENABLE_HDFS3=ON"
```

Comment:
After the pr https://github.com/facebookincubator/velox/commit/10cdf6fd8ca372b11779431ef77a177551c01700 , velox have removed VELOX_ENABLE_HDFS3